hw/chisel_acc/src/main/scala/snax_acc/reshuffle/Parameter.scala

```diff
@@ -0,0 +1,48 @@
+package snax_acc.reshuffle
+
+import chisel3._
+import chisel3.util._
+
+case class ReshufflerParams(
+    transposeInWidth: Int,
+    transposeOutWidth: Int,
+    inputWidth: Int,
+    outputWidth: Int,
+    poolLaneLen: Int,
+    readWriteCsrNum: Int,
+    sizeConfigWidth: Int,
+    tagName: String = "Reshuffler"
+)
+
+// parameters for Reshuffler accelerator
+object ReshufflerConstant {
+  def transposeInWidth = 512
+  def transposeOutWidth = 512
+
+  // input data type
+  def inputWidth = 8
+  // output data type
+  def outputWidth = 8
+
+  // Pooling parallelism
+  def poolLaneLen = 64
+
+  // we use 3 CSR for Reshuffler configuration,
```

Comment:
3 or 1? haha

Reply:
oh wait 1 RW for the transpose or not

then 2 for perf counter and 1 busy. Is it still worth it to have a perf counter and busy if you do this in one cycle?

Reply:
Yes!